src/Parsers/GeoCoordinateParser.php

```diff
@@ -113,7 +112,7 @@ public static function areCoordinates( $string ) {
 		static $parser = null;
 
 		if ( $parser === null ) {
-			$parser = new self( new ParserOptions() );
```

Comment:
Good question. I double-checked. The constructor parameter in the `StringValueParser` base class is optional since 0.1: https://github.com/DataValues/Common/blob/0.1/src/ValueParsers/StringValueParser.php#L29